cartography/data/jobs/analysis/aws_ec2_keypair_analysis.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "Analysis jobs for EC2 Key Pairs",
+    "statements": [
+        {
+            "__comment__": "Delete the attribute uploaded",
+            "query": "MATCH (k:EC2KeyPair) WHERE EXISTS (k.uploaded) REMOVE k.uploaded return COUNT(*) as TotalCompleted",
+            "iterative": false
+        },
+        {
+            "__comment__": "Delete the attribute duplicate_keyfingerprint",
+            "query": "MATCH (k:EC2KeyPair) WHERE EXISTS (k.duplicate_keyfingerprint) REMOVE k.duplicate_keyfingerprint return COUNT(*) as TotalCompleted",
+            "iterative": false
+        },
+        {
+            "__comment__": "Delete stale relationships between EC2KeyPairs",
+            "query": "MATCH (k1:EC2KeyPair)-[r:MATCHING_FINGERPRINT]-(k2:EC2KeyPair) WHERE r.lastupdated <> {UPDATE_TAG} DELETE (r) return COUNT(*) as TotalCompleted",
```

Comment:
Let's add directionality (a `<` or a `>`) in this query to restrict it a bit.

Reply:
Ok, so long as I don't add directionality in the `MERGE`, else it we'll get two relationships between the two nodes, instead of just one.